src/plugins/scene3d/Scene3D.cc

```diff
@@ -1058,6 +1060,30 @@ void IgnRenderer::BroadcastRightClick()
   App()->sendEvent(App()->findChild<MainWindow *>(), &rightClickToSceneEvent);
 }
 
+/////////////////////////////////////////////////
+void IgnRenderer::BroadcastKeyRelease()
+{
+  events::KeyRelease keyRelease(this->dataPtr->keyEvent);
```

Comment:
Nit: this can be moved into the `if`, right? No need to create the object if we aren't going to use it. The same below